### PR TITLE
REL-5086 Stress tests on bulk action of releases

### PR DESCRIPTION
**Pull request by jdolanski**
Bulk starting and aborting of releases inside release manager role

### CODE REVIEW

File: runner/src/main/scala/stress/chain/Folders.scala [hunk 1]
```diff
@@ -8,20 +8,28 @@ import stress.config.RunnerConfig
 
 object Folders {
 
+  val RELEASES_FILTER = s"""{"active":false,"planned":true,"completed":false,"onlyMine":false,"onlyFlagged":false,"filter":"","parentId":"Applications/Folder_1"}"""
```
Comment (byaminov) [inline runner/src/main/scala/stress/chain/Folders.scala]: Let's replace `filter` with `title` here and possible in other places: AFAIR the filtering name has changed in XLR some releases ago, you can see it in `ReleasesFilter.java`.

**Comment by byaminov** [comment]
Nice work @jdolanski 👍 (did not test it, but from quick code review perspective)

File: runner/src/main/scala/stress/chain/Tasks.scala [hunk 2]
```diff
@@ -4,16 +4,22 @@ import io.gatling.core.Predef._
...
 import scala.language.postfixOps
 
 object Tasks {
 
-  val ALL_TASKS_FILTER = """{"active":false,"assignedToMe":true,"assignedToMyTeams":true,"assignedToOthers":true,"notAssigned":true,"filter":""}"""
-  val MY_TASKS_FILTER = """{"active":false,"assignedToMe":true,"assignedToMyTeams":false,"assignedToOthers":false,"notAssigned":false,"filter":""}"""
-  val NOT_EXISTING_TASKS_FILTER = """{"active":false,"assignedToMe":true,"assignedToMyTeams":true,"assignedToOthers":true,"notAssigned":true,"filter":"non-existing"}"""
+  val ALL_TASKS_FILTER = TaskSearchFilter(assignedToMe = true, assignedToMyTeams = true, assignedToOthers = true, notAssigned = true)
+  val MY_TASKS_FILTER = TaskSearchFilter(assignedToMe = true)
+  val NOT_EXISTING_TASKS_FILTER = TaskSearchFilter(
```
Comment (hilic) [inline runner/src/main/scala/stress/chain/Tasks.scala]: `NON_EXISTING_TASKS_FILTER`? 😄

File: runner/src/main/scala/stress/chain/Tasks.scala [hunk 3]
```diff
@@ -25,6 +31,11 @@ object Tasks {
           .findAll.optional
           .saveAs("taskIds")
       )
+      .check(
+        jsonPath("$['releaseTasks'][*]['id']")
+          .findAll.optional
+          .saveAs("taskReleaseIds")
```
Comment (hilic) [inline runner/src/main/scala/stress/chain/Tasks.scala]: extract as constant?

File: runner/src/main/scala/stress/utils/Converters.scala [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package stress.utils
+
+object Converters {
```
Comment (hilic) [inline runner/src/main/scala/stress/utils/Converters.scala]: Don't we have an `Ids` service in XLR that does something like this? So maybe rename to be inline with XLR?

**Pull request merged**